pages/news-and-blog.html

```diff
@@ -5,6 +5,7 @@
 banner_file: banner--news-media-lg.svg
 banner_file_mobile: banner--news-media-sm.svg
 social_image: social-card--news-and-media.png
+social_image_alt_text: "Navy blue graphic block with blue and yellow overlapping spotlight shapes"
```

Comment:
Suggested text edit:

"Graphic illustration of two spotlights projecting downward on a dark blue field. A bright yellow area is highlighted where the two spotlights overlap."

Reply:
@drew-usds all revised alt-text look good. Thank you.